tests/traits/test_with_timestamp.py

```diff
@@ -58,10 +59,11 @@ def test_get_timetamp_or_duration_day_of_month_end_of_month_skip(wait):
 
 
 @pytest.mark.usefixtures("wait")
+@freeze_time("2017-02-03 12:01:02")  # It's a Friday.
 def test_get_timetamp_or_duration_next_weekday(wait):
     wait2 = copy.deepcopy(wait)
-    now = datetime.datetime.now()
-    now_day = now.weekday() + 1
+    now = datetime.datetime.now(pytz.timezone('Europe/Paris'))
```

Comment:
All tests were prepared to run in Europe/Paris timezone, however the machine running the test is not always in that timzeone.